pkg/dynamicinstrumentation/codegen/c/maps.h

```diff
@@ -45,7 +45,7 @@ struct {
 // of collections such as slices so that they can later be referenced
 // when reading the values in the collection.
 struct {
-    __uint(type, BPF_MAP_TYPE_HASH);
+    __uint(type, BPF_MAP_TYPE_LRU_HASH);
```

Comment:
Why do you need an LRU? They don't really work the way you would expect them to. They can evict many entries, rather than just the least-recently-used ones you desire.